test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,19 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(12)).toBe('Fizz')
```

Comment:
Would be good to get a couple more test cases so you know each example given here its just a fluke